contracts/Escrow.sol

```diff
@@ -0,0 +1,275 @@
+pragma solidity 0.4.24;
+import "./HMTokenInterface.sol";
+import "./SafeMath.sol";
+
+contract Escrow {
+    using SafeMath for uint256;
+    event IntermediateStorage(string _url, string _hash);
+    enum EscrowStatuses { Launched, Pending, Partial, Paid, Complete, Cancelled }
+    EscrowStatuses private status;
+
+    address private reputationOracle;
+    address private recordingOracle;
+
+    uint256 private reputationOracleStake;
+    uint256 private recordingOracleStake;
+
+    address private canceler;
+    address private eip20;
+
+    string private manifestUrl;
+    string private manifestHash;
+
+    string private intermediateManifestUrl;
+    string private intermediateManifestHash;
```

Comment:
Shouldn't these be intermediateResultsUrl and intermediateResultsHash?

Reply:
The contract itself is just a copypaste of the old `hmt.sol` file. I can change these to the ones you suggested if that's what they are supposed to be.

Reply:
No let's keep to the copy/paste approach